cmd/sandbox-ce/cmd/prune.go

```diff
@@ -0,0 +1,19 @@
+package cmd
+
+import (
+	"context"
+
+	"github.com/smacker/superset-compose/cmd/sandbox-ce/compose"
+)
+
+type pruneCmd struct {
+	Command `name:"prune" short-description:"Prune"`
+}
+
+func (c *pruneCmd) Execute(args []string) error {
+	return compose.Run(context.Background(), "down")
```

Comment:
most probably we want `--volumes` here

Reply:
Addressed [here](https://github.com/smacker/superset-compose/pull/1/commits/54eab1be637ae71878dd9b31221d709de3632dc5).